src/test/kotlin/no/nav/familie/ef/sak/opplysninger/personopplysninger/mapper/GrunnlagsdataMapperTest.kt

```diff
@@ -0,0 +1,42 @@
+package no.nav.familie.ef.sak.opplysninger.personopplysninger.mapper
+
+import no.nav.familie.ef.sak.opplysninger.personopplysninger.domene.Fødsel
+import no.nav.familie.ef.sak.testutil.PdlTestdataHelper.fødested
+import no.nav.familie.ef.sak.testutil.PdlTestdataHelper.fødselsdato
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Test
+import java.time.LocalDate
+
+internal class GrunnlagsdataMapperTest {
+    @Test
+    fun `to forskjellige fødselsdatoer og fødesteder skal bli merget til to Fødsel-objekter`() {
```

Comment:
Trenger kanskje ikke teste med "to", men fint med test på manglende data? 
Kanskje en "solskinstest" med en Fødsel + en hvor noe data mangler?